vms/platformvm/txs/executor/camino_tx_executor.go

```diff
@@ -443,6 +452,162 @@ func (e *CaminoProposalTxExecutor) RewardValidatorTx(tx *txs.RewardValidatorTx)
 	return nil
 }
 
+func (e *CaminoStandardTxExecutor) DepositTx(tx *txs.DepositTx) error {
+	caminoGenesis, err := e.State.CaminoGenesisState()
+	if err != nil {
+		return err
+	}
+
+	if !caminoGenesis.LockModeBondDeposit {
+		return errWrongLockMode
+	}
+
+	if err := locked.VerifyLockMode(tx.Ins, tx.Outs, caminoGenesis.LockModeBondDeposit); err != nil {
+		return err
+	}
+
+	depositAmount, err := tx.DepositAmount()
+	if err != nil {
+		return err
+	}
+
+	depositOffer, err := e.State.GetDepositOffer(tx.DepositOfferID)
+	if err != nil {
+		return err
+	}
+
+	currentChainTime := e.State.GetTimestamp()
+
+	switch {
+	case depositOffer == nil:
+		return errDepositOfferNotFound
```

Comment:
This error will never be reached because this case is already covered by line 476 that will return `not found` error. I suggest to remove this check here and return the `errDepositOfferNotFound` on line 476. What do you think?